views/data-analysis/index.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+//******************************************************************************
+//                                 index.php
+// PHIS-SILEX
+// Copyright © INRA 2018
+// Creation date: 21 feb. 2019
+// Contact: arnaud.charleroy@inra.fr, anne.tireau@inra.fr, pascal.neveu@inra.fr
+//******************************************************************************
+
+use yii\helpers\Html;
+use yii\helpers\Url;
+use app\models\yiiModels\DataAnalysisAppSearch;
+
+/* @var $this yii\web\View */
+/* @var $searchModel app\models\yiiModels\DataAnalysisAppSearch; */
+/* @var $dataProvider yii\data\ActiveDataProvider */
+
+$this->title = Yii::t('app', '{n, plural, =1{Stat/Vizu Application} other{Stat/Vizu Applications}}', ['n' => 2]);
+$this->params['breadcrumbs'][] = $this->title;
+if (Yii::$app->session->hasFlash('scriptNotAvailable')) {
+    echo Html::tag("p", "Script not available", ["class" => "alert alert-danger"]);
+}
+
+echo Html::beginTag("div", ["class" => "data-analysis-index"]);
+echo Html::beginTag("div", ["class" => "row"]);
+foreach ($dataProvider as $function => $appInfo) {
+    if ($integrated) {
+        $appHref =  Url::to([
+                    "data-analysis/run-script/", 
+                    "function" => $function,
+                    "rpackage" => $appInfo[DataAnalysisAppSearch::R_PACKAGE_NAME]
+                    ]);
+    } else {
+        $appHref = $appInfo[DataAnalysisAppSearch::APP_INDEX_HREF];
+    }
+    echo Html::beginTag("div", ["class" => "col-sm-6 col-md-5"]);
+    echo Html::beginTag("div", ["class" => "thumbnail"]);
+    echo Html::beginTag("a", ["href" => $appHref]);
```

Comment:
Why don't you use the Html::img method ? 
Maybe something such as this example : 
`return Html::a(Html::img(Yii::getAlias('@img-url'). '/myimage.jpg'),['index'])`

Reply:
I added it.